docs/cubic/cubic.md

```diff
@@ -218,3 +226,32 @@ After all these steps, it makes sense to return your .bashrc to non-writable mod
 ```bash
 $ chmod -w ~/.bashrc
 ```
+
+# Project Directory Access Request
+
+Once you have access to CUBIC, you may need to start a project in a new directory. Visit [this wiki](https://cbica-wiki.uphs.upenn.edu/wiki/index.php/Research_Projects#Project_Creation_Request) for more, or follow along below.
+
+First you need to fill out the data management document available [here](https://cbica-wiki.uphs.upenn.edu/wiki/images/Project_data_use_template.doc). This document will ask you for a number of details about your project, including the data's source and estimates about how much disk space you will need over a 6 month, 12 month, and 24 month period, and the estimated lifespan of the data ( 🤷). You will also need to provide the CUBIC usernames for everyone you want to have read and/or write access to the project — getting this done ahead of time is strongly recommended because, as you can imagine, requesting changes after-the-fact can be a bother.
+
+Additionally, you will need to be familiar with:
+
+- Whether or not the data has an IRB associated with it and who has approval
+- Whether or not the data is the *definitive* source
+- Whether or not you have a data use agreement
+- What will happen to the data at the end of its expected lifespan on the cluster
+
+This document must be saved as a `.txt` file and before being submitted with your request.
+
+Finally, you will need approval from your PI (Ted). This involves sending an email to the PI with a written blurb to the effect of "Do you approve of this project folder request", to which the PI only needs to respond "Yes, approved". Once you've got this you can screenshot the conversation (include the date in frame) and save that as an image.
```

Comment:
If other labs are going to use this documentation we might want to take out Ted

Reply:
True